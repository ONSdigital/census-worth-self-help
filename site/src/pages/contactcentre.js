import React from "react"
import { graphql } from "gatsby"
import Layout from "../components/layout"
import PageTitle from "../components/pagetitle"
import TextBlock from "../components/textblock"
import TabList from "../components/tablist"
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
import { faPhone } from "@fortawesome/free-solid-svg-icons"

import BlockButton from "../components/blockbutton"
import Section from "../components/section"

// remove spaces as they do not work in uri format
export const sanitizePhoneNumber = number => number.replace(/\s/g, "")

export const getSuggestedEdges = edges => {
  return edges
    .filter(edge => {
      return (
        edge.node.frontmatter.tags &&
        edge.node.frontmatter.tags.includes("popular")
      )
    })
    .slice(0, 3)
}

export default ({ data }) => {
  let suggestedEdges = getSuggestedEdges(data.allMarkdownRemark.edges)

  let phoneNumber = "01234 56789"
  let contactCentreText = `We are available to help you with any queries that have not been resolved by the Self help Facility. Lines are open 9am to 5pm on weekdays.`

  if (data.markdownRemark) {
    phoneNumber = data.markdownRemark.frontmatter.contact_centre_number
    contactCentreText = data.markdownRemark.frontmatter.contact_centre_text
  }

  return (
    <Layout>
      <PageTitle>Census Field Support</PageTitle>

      <TextBlock>
        {contactCentreText}
      </TextBlock>
      {suggestedEdges.length > 0 && (
        <TabList title="HAVE YOU TRIED..." elements={suggestedEdges} />
      )}
      <Section>
        <a href={"tel:" + sanitizePhoneNumber(phoneNumber)}>
          <BlockButton
            icon={<FontAwesomeIcon icon={faPhone} />}
            title="Call Census field support"
            subtitle={phoneNumber}
          />
        </a>
      </Section>
    </Layout>
  )
}

export const query = graphql`
  query {
    markdownRemark(frontmatter: { contact_centre_wait_time: { ne: null } }) {
      frontmatter {
        contact_centre_number
        contact_centre_text
      }
    }

    allMarkdownRemark(
      sort: { fields: frontmatter___date, order: DESC }
      filter: { fields: { collection: { eq: "articles" } } }
    ) {
      totalCount
      edges {
        node {
          ...BaseArticleFields
        }
      }
    }
  }
`
