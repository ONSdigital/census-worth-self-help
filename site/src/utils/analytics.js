export default class Analytics {
  static trackEvent(category, action, name = "", value = "") {
    if (window._paq) {
      window._paq.push(["trackEvent", category, action, name, value])
    } else {
      console.debug(`Track event not set : ${category} - ${action} - ${name} - ${value}`)
    }
  }

  static trackSiteSearch(keyword, category, searchCount) {
    if (window._paq) {
      window._paq.push(["trackSiteSearch", keyword, category, searchCount])
    } else {
      console.debug(`Track site search : ${keyword} - ${category} - ${searchCount}`)
    }
  }

  static setPageType(pageType) {
    if (typeof window !== `undefined` && window._paq) {
      window._paq.push(["setCustomDimension", 1, pageType])
    }
  }
}