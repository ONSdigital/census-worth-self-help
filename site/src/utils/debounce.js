import _ from "lodash"

// debounce with default delay, or return sync function if delay exactly zero (for unit tests)
export default (fn, delay) => {
  return (delay === 0)
    ? fn
    : _.debounce((...args) => fn(...args), delay || 1000)
}