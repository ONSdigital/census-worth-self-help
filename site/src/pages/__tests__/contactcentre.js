import React from "react"
import renderer from "react-test-renderer"
import ContactCentre from "../contactcentre"
import { sanitizePhoneNumber } from "../contactcentre"
import { render } from "react-testing-library"
import { articleList } from "../../utils/testdata"

describe("Contact centre", () => {
  it("renders correctly without data", () => {
    const data = {allMarkdownRemark : { edges : []}}
    const tree = renderer.create(<ContactCentre data={data}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly with data", () => {
    const data = { allMarkdownRemark : articleList,
    	markdownRemark : { frontmatter: {contact_centre_wait_time: "25", contact_centre_number: "01111 333333" }} }
    const tree = renderer.create(<ContactCentre data={data}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("phone numbers are space escaped", () => {
    expect(sanitizePhoneNumber("0123 456 789")).toEqual("0123456789")
  })
})