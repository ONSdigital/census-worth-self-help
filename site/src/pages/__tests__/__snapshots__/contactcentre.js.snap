// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact centre renders correctly with data 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "vkixka",
      "next": undefined,
      "styles": "
        height: 100vh;
        display: flex;
        flex-direction: column;
      ",
    }
  }
>
  <header
    css={
      Object {
        "map": undefined,
        "name": "181ra4m",
        "next": undefined,
        "styles": "
        width: 100vw;
        
    background: linear-gradient(to left, rgb(0,61,89), #33637a);
  ;
      ",
      }
    }
  >
    <div
      css={
        Object {
          "map": undefined,
          "name": "1thgah9",
          "next": undefined,
          "styles": "
            display: flex;
          ",
        }
      }
    >
      <div
        className="Header-Title-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "17ih1bc",
            "next": undefined,
            "styles": "
              
    display: flex;
    justify-content: center;
    flex-direction: column;
  
              
    padding: 15px;
  
              flex-grow: 1;
            ",
          }
        }
        onClick={[Function]}
      >
        Self Help Facility
      </div>
      <div
        className="Header-button-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "1sm5jo",
            "next": undefined,
            "styles": "
        
    display: flex;
    justify-content: center;
    flex-direction: column;
  
        padding: 0px 10px;
        background-color: rgb(255, 255, 255, 0.2);
        border-radius: 4px;
        margin: 10px 10px 10px 0px;
        min-width: 20px;
        object-fit: contain;
        text-align: center;
        height: 40px;
      ",
          }
        }
        data-testid="topbarlink-button"
        onClick={[Function]}
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "1dbjmka",
              "next": undefined,
              "styles": "
                  font-size: 19px;
                ",
            }
          }
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-search fa-w-16 "
            data-icon="search"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </div>
      </div>
      <div
        className="Header-button-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "1sm5jo",
            "next": undefined,
            "styles": "
        
    display: flex;
    justify-content: center;
    flex-direction: column;
  
        padding: 0px 10px;
        background-color: rgb(255, 255, 255, 0.2);
        border-radius: 4px;
        margin: 10px 10px 10px 0px;
        min-width: 20px;
        object-fit: contain;
        text-align: center;
        height: 40px;
      ",
          }
        }
        data-testid="topbarlink-button"
        onClick={[Function]}
      >
        Menu
      </div>
    </div>
  </header>
  <div
    css={
      Object {
        "map": undefined,
        "name": "o9ztju",
        "next": undefined,
        "styles": "
          
    margin: 0 auto;
    max-width: 700px;
  
          overflow: scroll;
          width: 100vw;
          display: flex;
          flex-direction: column;
          flex-grow: 1;
        ",
      }
    }
  >
    <main
      css={
        Object {
          "map": undefined,
          "name": "1k0aln6",
          "next": undefined,
          "styles": "
            flex-grow: 1;
          ",
        }
      }
    >
      <div
        css={
          Object {
            "map": undefined,
            "name": "12ew49b",
            "next": undefined,
            "styles": "
        
    padding: 5px 15px;
  
      ",
          }
        }
      >
        <h1
          className="Article-Title-Style"
          css={
            Object {
              "map": undefined,
              "name": "1xcgh0k",
              "next": undefined,
              "styles": "
          margin-bottom: 0px;
          display: flex;
        ",
            }
          }
          data-testid="search-result-title"
        >
          Census Field Support
        </h1>
      </div>
      <div
        className="Article-body-Style"
        css={
          Object {
            "map": undefined,
            "name": "bsbyq4",
            "next": undefined,
            "styles": "
        
    padding: 5px 15px;
  ;
        margin-bottom: 20px;
      ",
          }
        }
      >
        We are available to help you with any queries that have not been resolved by the Self help Facility. Lines are open 8am to 5pm on weekdays.
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "1p7jpnp",
            "next": undefined,
            "styles": "
        
    padding: 15px 0px;
  ;
      ",
          }
        }
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "2artsq",
              "next": undefined,
              "styles": "
          
    padding: 15px 0px;
  ;
          
    padding: 5px 15px;
  
          display: flex;
          margin-bottom: 5px;
        ",
            }
          }
        >
          <div
            className="Tablist-heading-Style"
          >
            HAVE YOU TRIED...
          </div>
        </div>
        <div>
          <div
            className="clickable"
            css={
              Object {
                "map": undefined,
                "name": "lcxyxv",
                "next": undefined,
                "styles": "
        
    padding: 15px;
    min-height: 30px;
  ;
        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.5);
        border-left: 6px solid rgb(0,61,89);
        background-color: rgb(255, 255, 255);
      ",
              }
            }
            onClick={[Function]}
          >
            <div
              className="Card-heading-Style"
              css={
                Object {
                  "map": undefined,
                  "name": "qiy98z",
                  "next": undefined,
                  "styles": "
          color: inherit;
          flex-grow: 1;
          font-weight: semi-bold;
        ",
                }
              }
              data-testid="articletab-article-card"
            >
              test Article 2
            </div>
            <div
              className="Card-sub-head-Style-gray"
              css={
                Object {
                  "map": undefined,
                  "name": "1hejrb1",
                  "next": undefined,
                  "styles": "
          padding-top: 6px;
          display: flex;
          flex-direction: vertical;
          line-height: 1.08;
          max-height: 42.12px; // line-height * font-size * 3
        ",
                }
              }
            >
              <div
                css={
                  Object {
                    "map": undefined,
                    "name": "lz6rtp",
                    "next": undefined,
                    "styles": "
            width: 70%;
            overflow: hidden;
            text-overflow: ellipsis;
            flex-grow: 1;
          ",
                  }
                }
              >
                testDescription
              </div>
              <div
                className="Card-meta-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "4j3tb8",
                    "next": undefined,
                    "styles": "
            display: flex;
            flex-direction: column;
            justify-content: flex-end;
          ",
                  }
                }
              >
                a few seconds ago
              </div>
            </div>
          </div>
        </div>
        <div>
          <div
            className="clickable"
            css={
              Object {
                "map": undefined,
                "name": "lcxyxv",
                "next": undefined,
                "styles": "
        
    padding: 15px;
    min-height: 30px;
  ;
        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.5);
        border-left: 6px solid rgb(0,61,89);
        background-color: rgb(255, 255, 255);
      ",
              }
            }
            onClick={[Function]}
          >
            <div
              className="Card-heading-Style"
              css={
                Object {
                  "map": undefined,
                  "name": "qiy98z",
                  "next": undefined,
                  "styles": "
          color: inherit;
          flex-grow: 1;
          font-weight: semi-bold;
        ",
                }
              }
              data-testid="articletab-article-card"
            >
              test Article 3
            </div>
            <div
              className="Card-sub-head-Style-gray"
              css={
                Object {
                  "map": undefined,
                  "name": "1hejrb1",
                  "next": undefined,
                  "styles": "
          padding-top: 6px;
          display: flex;
          flex-direction: vertical;
          line-height: 1.08;
          max-height: 42.12px; // line-height * font-size * 3
        ",
                }
              }
            >
              <div
                css={
                  Object {
                    "map": undefined,
                    "name": "lz6rtp",
                    "next": undefined,
                    "styles": "
            width: 70%;
            overflow: hidden;
            text-overflow: ellipsis;
            flex-grow: 1;
          ",
                  }
                }
              >
                testDescription
              </div>
              <div
                className="Card-meta-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "4j3tb8",
                    "next": undefined,
                    "styles": "
            display: flex;
            flex-direction: column;
            justify-content: flex-end;
          ",
                  }
                }
              >
                a few seconds ago
              </div>
            </div>
          </div>
        </div>
        <div>
          <div
            className="clickable"
            css={
              Object {
                "map": undefined,
                "name": "lcxyxv",
                "next": undefined,
                "styles": "
        
    padding: 15px;
    min-height: 30px;
  ;
        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.5);
        border-left: 6px solid rgb(0,61,89);
        background-color: rgb(255, 255, 255);
      ",
              }
            }
            onClick={[Function]}
          >
            <div
              className="Card-heading-Style"
              css={
                Object {
                  "map": undefined,
                  "name": "qiy98z",
                  "next": undefined,
                  "styles": "
          color: inherit;
          flex-grow: 1;
          font-weight: semi-bold;
        ",
                }
              }
              data-testid="articletab-article-card"
            >
              test Article 5
            </div>
            <div
              className="Card-sub-head-Style-gray"
              css={
                Object {
                  "map": undefined,
                  "name": "1hejrb1",
                  "next": undefined,
                  "styles": "
          padding-top: 6px;
          display: flex;
          flex-direction: vertical;
          line-height: 1.08;
          max-height: 42.12px; // line-height * font-size * 3
        ",
                }
              }
            >
              <div
                css={
                  Object {
                    "map": undefined,
                    "name": "lz6rtp",
                    "next": undefined,
                    "styles": "
            width: 70%;
            overflow: hidden;
            text-overflow: ellipsis;
            flex-grow: 1;
          ",
                  }
                }
              >
                testDescription
              </div>
              <div
                className="Card-meta-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "4j3tb8",
                    "next": undefined,
                    "styles": "
            display: flex;
            flex-direction: column;
            justify-content: flex-end;
          ",
                  }
                }
              >
                a few seconds ago
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "1p7jpnp",
            "next": undefined,
            "styles": "
        
    padding: 15px 0px;
  ;
      ",
          }
        }
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "2artsq",
              "next": undefined,
              "styles": "
          
    padding: 15px 0px;
  ;
          
    padding: 5px 15px;
  
          display: flex;
          margin-bottom: 5px;
        ",
            }
          }
        >
          <div
            className="Tablist-heading-Style"
          >
            CONTACTING FIELD SUPPORT
          </div>
        </div>
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "bn78dc",
            "next": undefined,
            "styles": "
        box-shadow: 0 2px 4px 0 rgba(212, 212, 212, 0.5);
        border: solid 1px #ececed;
        background-color: rgb(255, 255, 255);
        height: 74px;
        display: flex;
        align-items: center;
        justify-content: center;
      ",
          }
        }
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "1w87xe6",
              "next": undefined,
              "styles": "
          color: rgb(0,61,89);
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          font-size: 30px;
          flex: 1 1 0;
          align-items: center;
          padding: 0px 5px;
        ",
            }
          }
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-clock fa-w-16 "
            data-icon="clock"
            data-prefix="far"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm61.8-104.4l-84.9-61.7c-3.1-2.3-4.9-5.9-4.9-9.7V116c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v141.7l66.8 48.6c5.4 3.9 6.5 11.4 2.6 16.8L334.6 349c-3.9 5.3-11.4 6.5-16.8 2.6z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </div>
        <div
          css={
            Object {
              "map": undefined,
              "name": "sj34c4",
              "next": undefined,
              "styles": "
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          text-align: center;
        ",
            }
          }
        >
          <div
            className="Card-heading-Style"
            css={
              Object {
                "map": undefined,
                "name": "2e2jvx",
                "next": undefined,
                "styles": "
            margin-bottom: 5px;
          ",
              }
            }
          >
            25 minutes
          </div>
          <div
            className="Card-sub-head-Style-gray"
          >
            Current waiting time
          </div>
        </div>
        <div
          css={
            Object {
              "map": undefined,
              "name": "531ogy",
              "next": undefined,
              "styles": "
          flex: 1 1 0;
        ",
            }
          }
        />
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "ypl6yy",
            "next": undefined,
            "styles": "
        border-bottom: 1px rgb(229, 229, 234) solid;
        padding: 15px 0px;
      ",
          }
        }
      >
        <a
          href="tel:01111333333"
        >
          <div
            className="clickable"
            css={
              Object {
                "map": undefined,
                "name": "1vf6ctb",
                "next": undefined,
                "styles": "
        border-radius: 4px;
        box-shadow: 0 2px 4px 0 rgba(212, 212, 212, 0.5);
        border: solid 1px #ececed;
        margin: 0px 10px;
        background-color: rgb(255, 255, 255);
        display: flex;
      ",
              }
            }
            data-testid="block-button"
          >
            <div
              className="Article-Title-Style"
              css={
                Object {
                  "map": undefined,
                  "name": "p10n1l",
                  "next": undefined,
                  "styles": "
            
    display: flex;
    justify-content: center;
    flex-direction: column;
  
            display: flex;
            justify-content: center;
            height: 78px;
            width: 78px;
            color: white;
            background-color: rgb(0,61,89);
          ",
                }
              }
            >
              <div
                css={
                  Object {
                    "map": undefined,
                    "name": "1n8pnzb",
                    "next": undefined,
                    "styles": "
              text-align: center;
            ",
                  }
                }
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-phone fa-w-16 "
                  data-icon="phone"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </div>
            </div>
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1o8j1iv",
                  "next": undefined,
                  "styles": "
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          margin: 0px 20px;
          color: rgb(0,61,89);
        ",
                }
              }
            >
              <div
                className="Button-heading-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "km4e7l",
                    "next": undefined,
                    "styles": "
            text-align: left;
          ",
                  }
                }
              >
                Call Census field support
              </div>
              <div
                className="Card-meta-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "km4e7l",
                    "next": undefined,
                    "styles": "
            text-align: left;
          ",
                  }
                }
              >
                01111 333333
              </div>
            </div>
          </div>
        </a>
      </div>
    </main>
    <div
      css={
        Object {
          "map": undefined,
          "name": "ej75ms",
          "next": undefined,
          "styles": "
        
    padding: 30px 0px;
  
        width: 100%;
        padding: 10px 0px;
        background-color: rgb(236, 236, 237);
        display: flex;
        min-height: 100px;
        flex-direction: column;
        justify-content: center;
        flex-shrink: 0;
      ",
        }
      }
    >
      <img
        alt="Footer logo"
        css={
          Object {
            "map": undefined,
            "name": "1v6eegl",
            "next": undefined,
            "styles": "
          margin-bottom: 50px;
          margin-top: 50px;
        ",
          }
        }
        src="/footer_logo.svg"
      />
      <div
        className="Footer-Style"
      >
        © Office for National Statistics 2019
      </div>
    </div>
  </div>
</div>
`;

exports[`Contact centre renders correctly without data 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "vkixka",
      "next": undefined,
      "styles": "
        height: 100vh;
        display: flex;
        flex-direction: column;
      ",
    }
  }
>
  <header
    css={
      Object {
        "map": undefined,
        "name": "181ra4m",
        "next": undefined,
        "styles": "
        width: 100vw;
        
    background: linear-gradient(to left, rgb(0,61,89), #33637a);
  ;
      ",
      }
    }
  >
    <div
      css={
        Object {
          "map": undefined,
          "name": "1thgah9",
          "next": undefined,
          "styles": "
            display: flex;
          ",
        }
      }
    >
      <div
        className="Header-Title-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "17ih1bc",
            "next": undefined,
            "styles": "
              
    display: flex;
    justify-content: center;
    flex-direction: column;
  
              
    padding: 15px;
  
              flex-grow: 1;
            ",
          }
        }
        onClick={[Function]}
      >
        Self Help Facility
      </div>
      <div
        className="Header-button-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "1sm5jo",
            "next": undefined,
            "styles": "
        
    display: flex;
    justify-content: center;
    flex-direction: column;
  
        padding: 0px 10px;
        background-color: rgb(255, 255, 255, 0.2);
        border-radius: 4px;
        margin: 10px 10px 10px 0px;
        min-width: 20px;
        object-fit: contain;
        text-align: center;
        height: 40px;
      ",
          }
        }
        data-testid="topbarlink-button"
        onClick={[Function]}
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "1dbjmka",
              "next": undefined,
              "styles": "
                  font-size: 19px;
                ",
            }
          }
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-search fa-w-16 "
            data-icon="search"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </div>
      </div>
      <div
        className="Header-button-Style clickable"
        css={
          Object {
            "map": undefined,
            "name": "1sm5jo",
            "next": undefined,
            "styles": "
        
    display: flex;
    justify-content: center;
    flex-direction: column;
  
        padding: 0px 10px;
        background-color: rgb(255, 255, 255, 0.2);
        border-radius: 4px;
        margin: 10px 10px 10px 0px;
        min-width: 20px;
        object-fit: contain;
        text-align: center;
        height: 40px;
      ",
          }
        }
        data-testid="topbarlink-button"
        onClick={[Function]}
      >
        Menu
      </div>
    </div>
  </header>
  <div
    css={
      Object {
        "map": undefined,
        "name": "o9ztju",
        "next": undefined,
        "styles": "
          
    margin: 0 auto;
    max-width: 700px;
  
          overflow: scroll;
          width: 100vw;
          display: flex;
          flex-direction: column;
          flex-grow: 1;
        ",
      }
    }
  >
    <main
      css={
        Object {
          "map": undefined,
          "name": "1k0aln6",
          "next": undefined,
          "styles": "
            flex-grow: 1;
          ",
        }
      }
    >
      <div
        css={
          Object {
            "map": undefined,
            "name": "12ew49b",
            "next": undefined,
            "styles": "
        
    padding: 5px 15px;
  
      ",
          }
        }
      >
        <h1
          className="Article-Title-Style"
          css={
            Object {
              "map": undefined,
              "name": "1xcgh0k",
              "next": undefined,
              "styles": "
          margin-bottom: 0px;
          display: flex;
        ",
            }
          }
          data-testid="search-result-title"
        >
          Census Field Support
        </h1>
      </div>
      <div
        className="Article-body-Style"
        css={
          Object {
            "map": undefined,
            "name": "bsbyq4",
            "next": undefined,
            "styles": "
        
    padding: 5px 15px;
  ;
        margin-bottom: 20px;
      ",
          }
        }
      >
        We are available to help you with any queries that have not been resolved by the Self help Facility. Lines are open 8am to 5pm on weekdays.
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "1p7jpnp",
            "next": undefined,
            "styles": "
        
    padding: 15px 0px;
  ;
      ",
          }
        }
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "2artsq",
              "next": undefined,
              "styles": "
          
    padding: 15px 0px;
  ;
          
    padding: 5px 15px;
  
          display: flex;
          margin-bottom: 5px;
        ",
            }
          }
        >
          <div
            className="Tablist-heading-Style"
          >
            CONTACTING FIELD SUPPORT
          </div>
        </div>
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "bn78dc",
            "next": undefined,
            "styles": "
        box-shadow: 0 2px 4px 0 rgba(212, 212, 212, 0.5);
        border: solid 1px #ececed;
        background-color: rgb(255, 255, 255);
        height: 74px;
        display: flex;
        align-items: center;
        justify-content: center;
      ",
          }
        }
      >
        <div
          css={
            Object {
              "map": undefined,
              "name": "1w87xe6",
              "next": undefined,
              "styles": "
          color: rgb(0,61,89);
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          font-size: 30px;
          flex: 1 1 0;
          align-items: center;
          padding: 0px 5px;
        ",
            }
          }
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-clock fa-w-16 "
            data-icon="clock"
            data-prefix="far"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm61.8-104.4l-84.9-61.7c-3.1-2.3-4.9-5.9-4.9-9.7V116c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v141.7l66.8 48.6c5.4 3.9 6.5 11.4 2.6 16.8L334.6 349c-3.9 5.3-11.4 6.5-16.8 2.6z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </div>
        <div
          css={
            Object {
              "map": undefined,
              "name": "sj34c4",
              "next": undefined,
              "styles": "
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          text-align: center;
        ",
            }
          }
        >
          <div
            className="Card-heading-Style"
            css={
              Object {
                "map": undefined,
                "name": "2e2jvx",
                "next": undefined,
                "styles": "
            margin-bottom: 5px;
          ",
              }
            }
          >
            15 minutes
          </div>
          <div
            className="Card-sub-head-Style-gray"
          >
            Current waiting time
          </div>
        </div>
        <div
          css={
            Object {
              "map": undefined,
              "name": "531ogy",
              "next": undefined,
              "styles": "
          flex: 1 1 0;
        ",
            }
          }
        />
      </div>
      <div
        css={
          Object {
            "map": undefined,
            "name": "ypl6yy",
            "next": undefined,
            "styles": "
        border-bottom: 1px rgb(229, 229, 234) solid;
        padding: 15px 0px;
      ",
          }
        }
      >
        <a
          href="tel:0123456789"
        >
          <div
            className="clickable"
            css={
              Object {
                "map": undefined,
                "name": "1vf6ctb",
                "next": undefined,
                "styles": "
        border-radius: 4px;
        box-shadow: 0 2px 4px 0 rgba(212, 212, 212, 0.5);
        border: solid 1px #ececed;
        margin: 0px 10px;
        background-color: rgb(255, 255, 255);
        display: flex;
      ",
              }
            }
            data-testid="block-button"
          >
            <div
              className="Article-Title-Style"
              css={
                Object {
                  "map": undefined,
                  "name": "p10n1l",
                  "next": undefined,
                  "styles": "
            
    display: flex;
    justify-content: center;
    flex-direction: column;
  
            display: flex;
            justify-content: center;
            height: 78px;
            width: 78px;
            color: white;
            background-color: rgb(0,61,89);
          ",
                }
              }
            >
              <div
                css={
                  Object {
                    "map": undefined,
                    "name": "1n8pnzb",
                    "next": undefined,
                    "styles": "
              text-align: center;
            ",
                  }
                }
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-phone fa-w-16 "
                  data-icon="phone"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </div>
            </div>
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1o8j1iv",
                  "next": undefined,
                  "styles": "
          
    display: flex;
    justify-content: center;
    flex-direction: column;
  
          margin: 0px 20px;
          color: rgb(0,61,89);
        ",
                }
              }
            >
              <div
                className="Button-heading-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "km4e7l",
                    "next": undefined,
                    "styles": "
            text-align: left;
          ",
                  }
                }
              >
                Call Census field support
              </div>
              <div
                className="Card-meta-Style"
                css={
                  Object {
                    "map": undefined,
                    "name": "km4e7l",
                    "next": undefined,
                    "styles": "
            text-align: left;
          ",
                  }
                }
              >
                01234 56789
              </div>
            </div>
          </div>
        </a>
      </div>
    </main>
    <div
      css={
        Object {
          "map": undefined,
          "name": "ej75ms",
          "next": undefined,
          "styles": "
        
    padding: 30px 0px;
  
        width: 100%;
        padding: 10px 0px;
        background-color: rgb(236, 236, 237);
        display: flex;
        min-height: 100px;
        flex-direction: column;
        justify-content: center;
        flex-shrink: 0;
      ",
        }
      }
    >
      <img
        alt="Footer logo"
        css={
          Object {
            "map": undefined,
            "name": "1v6eegl",
            "next": undefined,
            "styles": "
          margin-bottom: 50px;
          margin-top: 50px;
        ",
          }
        }
        src="/footer_logo.svg"
      />
      <div
        className="Footer-Style"
      >
        © Office for National Statistics 2019
      </div>
    </div>
  </div>
</div>
`;
